푸터-마일드/layout/basic/css/layout-new.css

```
@@ -0,0 +1,277 @@
+#header .inner .top_nav_box .top_category .sub_cate01 li a,
+#header .inner .top_nav_box .top_category .sub-category li a {display:flex;justify-content:center;align-items:center;text-align:center;min-width:110px;height:100%;font-size:14px;color:#999;line-height:1;font-weight:300;padding:0 10px;}
+#header .inner .top_nav_box .top_category .sub_cate01 li:hover > a,
+#header .inner .top_nav_box .top_category .sub-category li:hover > a {background-color:#f4f4f4;color:#333;}
+#header .inner .top_nav_box .top_mypage {margin-left:auto;display:flex;flex-shrink:0;height:30px;}
+#header .inner .top_nav_box .top_mypage img {width: 30px; height: auto;}
+#header .inner .top_nav_box .top_mypage a {font-size:16px;font-weight:400;line-height:1;color:#1a1a1a;margin: 0 14.5px;padding:0;border:0;display:flex;align-items:center;position:relative;}
+#header .inner .top_nav_box .top_mypage a.xans-layout-orderbasketcount .count {position:absolute;top:13px;right:-8px;width:18px;height:18px;border-radius:50%;background-color:#000;color:#fff;text-align:center;line-height:1;font-size:11px;display:flex;align-items:center;justify-content:center;margin-left:5px;font-family:Arial, sans-serif !important;}
+#header .inner .top_nav_box .top_mypage a.eNavFold {display:none;}
+#header .inner .top_nav_box .top_mypage div.mypage {position: relative;}
+#header .inner .top_nav_box .top_mypage .tablet_detail_only {display: none;}
+#header.fixed {position:fixed;top:0;left:0;right:0;z-index:998;margin:0 auto;-webkist-transform:translateY(-50px);-ms-transform:translateY(-50px);transform:translateY(-50px);}
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)